scripts/environment/bootstrap-ubuntu-20.04.sh

```diff
@@ -130,13 +136,13 @@ tar \
 cp "${TEMP}/${MOLD_TARGET}/bin/mold" /usr/bin/mold
 
 # Set Cargo to use mold as its linker.
-CARGO_OVERRIDE_DIR="${HOME}/.csh check for exisargo"
+CARGO_OVERRIDE_DIR="${HOME}/.cargo"
 mkdir -p "${CARGO_OVERRIDE_DIR}"
 
 CARGO_OVERRIDE_CONF="${CARGO_OVERRIDE_DIR}/config.toml"
-cat <<EOF >"${CARGO_OVERRIDE_CONF}"
+cat <<EOF >>"${CARGO_OVERRIDE_CONF}"
 [target.x86_64-unknown-linux-gnu]
-rustflags = ["-C", "linker=clang", "-C", "link-arg=-fuse-ld=/usr/bin/mold"]
+rustflags = ["-D", "warnings", "-C", "linker=clang", "-C", "link-arg=-fuse-ld=/usr/bin/mold"]
```

Comment:
Do we need to update the rustflags for other targets here as well?

Reply:
We only run our primary CI suite against x86-64 Linux, so Bruce is just mirroring his removal of `-D warnings` from the `RUSTFLAGS` environment variable we were setting directly in `.github/workflows/test.yaml`, etc.